pkg/auth/auth.go

```diff
@@ -369,21 +369,21 @@ func (a *Authenticator) CallbackFunc(fn func(loginInfo LoginJSON, successURL str
 		}
 
 		if code == "" {
-			klog.Info("missing auth code in query param")
+			klog.Error("missing auth code in query param")
 			a.redirectAuthError(w, errorMissingCode)
 			return
 		}
 
 		if urlState != cookieState.Value {
-			klog.Error("State in url does not match State cookie")
+			klog.Error("state in url does not match State cookie")
 			a.redirectAuthError(w, errorInvalidState)
 			return
 		}
 		ctx := oidc.ClientContext(context.TODO(), a.clientFunc())
 		oauthConfig, lm := a.authFunc()
 		token, err := oauthConfig.Exchange(ctx, code)
 		if err != nil {
-			klog.Infof("unable to verify auth code with issuer: %v", err)
+			klog.Errorf("unable to verify auth code with issuer: %v", err)
```

Comment:
This one is not expected as well.

Reply:
then it should be logged by default, right?